cli/tests/integrations/common.py

```diff
@@ -380,6 +417,25 @@ def app(path, app_id, deploy=False):
 
 
 @contextlib.contextmanager
+def package(package_name, deploy=False, args=[]):
+    """Context manager that deploys an app on entrance, and removes it on
+    exit.
+
+    :param package_name: package name
+    :type package_name: str
+    :param deploy: If True, block on the deploy
+    :type deploy: bool
+    :rtype: None
+    """
+
+    package_install(package_name, deploy, args)
```

Comment:
It's a context manager.  We only `try` below so we can ensure the package is uninstalled if the manager's block throws an exception